analyzers/src/SonarAnalyzer.CSharp/Rules/DatabasePasswordsShouldBeSecure.cs

```diff
@@ -68,7 +60,7 @@ public DatabasePasswordsShouldBeSecure()
         protected override void Initialize(TrackerInput input)
```

Comment:
One really NITPICK comment: The Language  property from line 58 should go before the constructor